grpc/tests/src/test/regression/ee_1103.rs

```diff
@@ -0,0 +1,324 @@
+use lazy_static::lazy_static;
+
+use casper_engine_test_support::{
+    internal::{utils, ExecuteRequestBuilder, InMemoryWasmTestBuilder, DEFAULT_ACCOUNTS},
+    MINIMUM_ACCOUNT_CREATION_BALANCE,
+};
+use casper_execution_engine::{core::engine_state::GenesisAccount, shared::motes::Motes};
+use casper_types::{
+    account::AccountHash,
+    auction::{ARG_DELEGATOR, ARG_VALIDATOR},
+    runtime_args, PublicKey, RuntimeArgs, U512,
+};
+
+const ARG_TARGET: &str = "target";
+const ARG_AMOUNT: &str = "amount";
+
+const CONTRACT_TRANSFER_TO_ACCOUNT: &str = "transfer_to_account_u512.wasm";
+const CONTRACT_DELEGATE: &str = "delegate.wasm";
+const TRANSFER_AMOUNT: u64 = MINIMUM_ACCOUNT_CREATION_BALANCE;
+const SYSTEM_ADDR: AccountHash = AccountHash::new([0u8; 32]);
+
+const FAUCET: PublicKey = PublicKey::Ed25519([1; 32]);
+const VALIDATOR_1: PublicKey = PublicKey::Ed25519([3; 32]);
+const VALIDATOR_2: PublicKey = PublicKey::Ed25519([5; 32]);
+const VALIDATOR_3: PublicKey = PublicKey::Ed25519([7; 32]);
+const DELEGATOR_1: PublicKey = PublicKey::Ed25519([204; 32]);
+const DELEGATOR_2: PublicKey = PublicKey::Ed25519([206; 32]);
+const DELEGATOR_3: PublicKey = PublicKey::Ed25519([208; 32]);
+
+lazy_static! {
+    static ref FAUCET_ADDR: AccountHash = FAUCET.into();
+    static ref VALIDATOR_1_ADDR: AccountHash = VALIDATOR_1.into();
+    static ref VALIDATOR_2_ADDR: AccountHash = VALIDATOR_2.into();
+    static ref VALIDATOR_3_ADDR: AccountHash = VALIDATOR_3.into();
+    static ref FAUCET_BALANCE: U512 = U512::from(100_000_000_000_000_000u64);
+    static ref VALIDATOR_1_BALANCE: U512 = U512::from(1_000_000_000_000_000u64);
+    static ref VALIDATOR_2_BALANCE: U512 = U512::from(1_000_000_000_000_000u64);
+    static ref VALIDATOR_3_BALANCE: U512 = U512::from(1_000_000_000_000_000u64);
+    static ref VALIDATOR_1_STAKE: U512 = U512::from(500_000_000_000_000u64);
+    static ref VALIDATOR_2_STAKE: U512 = U512::from(400_000_000_000_000u64);
+    static ref VALIDATOR_3_STAKE: U512 = U512::from(300_000_000_000_000u64);
+    static ref DELEGATOR_1_ADDR: AccountHash = DELEGATOR_1.into();
+    static ref DELEGATOR_2_ADDR: AccountHash = DELEGATOR_2.into();
+    static ref DELEGATOR_3_ADDR: AccountHash = DELEGATOR_3.into();
+    static ref DELEGATOR_1_BALANCE: U512 = U512::from(1_000_000_000_000_000u64);
```

Comment:
I.e. here a `MINIMUM_ACCOUNT_CREATION_BALANCE` would be better as those are accounts created with a transfer method. I hope changing this won't affect the test logic

Reply:
That will not work here because the balance of the account will be the same as its intended bid amount, so the call to delegate won't have enough $ to cover gas costs.